config/config.exs

```diff
@@ -2,6 +2,8 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
+config :my_app, Repo,
+  pool: Ecto.Adapters.SQL.Sandbox
```

Comment:
i don't think this should be in here. `config/test.exs` should be enough. the rest looks good, though.